_posts/2020/2020-03-10-working-as-a-scrum-master-at-epages.md

```diff
@@ -0,0 +1,113 @@
+In this internship I got a deeper insight into Scrum and agile work which I really liked.
+
+## What are your tasks and how many teams do you support? 
+
+My task area is very diverse.
+A big part of my work is collaborating with the Scrum teams.
+Currently, I'm supporting two teams in Jena.
+But these teams also include team members who work remotely from Bratislava and Leipzig.
+My aim is to ensure that my teams can work undisturbed in a harmonic environment without any conflicts.
+If there are any conflicts, I try to adress them, and solve them as fast as possible.
+
+Organisational development is a big task as well, which includes organizing and hosting e.g. the Daily Scrum or Retrospectives.
+There are some important aspects I have to consider as a Scrum Master: listen to the team, meet their needs, and building trust within the teams.
+But my work is not only limited to working with the Scrum teams.
+The Scrum Master's objective is to encourage team work also across different locations.
+Our task is for example to coordinate that the teams work cross-functional, to handle communication bareers, to avoid miscommunications between the individual teams, and to support the teams in order to work better together.
+
+We do not only work with the RnD teams, but also with other teams like Project Management, Marketing, HR, and Sales in order to develop new team processes.
+Thus, we would like to ensure that the whole company moves into the same direction.
+
+## Could you tell us how your typical work day looks like?
+
+Actually, each day as a Scrum Master is different.
+You never know what the day will bring you. 
+So, there's no typical work day, and that's what I like a lot about my job.
+But I will try to explain how my "typical" work day looks like.
+
+* **Morning:**
+My day starts at 7.30 a.m. at ePages.
+First, I get a coffee and try to exchange with other colleagues in the kitchen to keep me updated.
+Then, I return to my desk to check my emails and messages.
+Afterwards, I priotize my tasks for the day to get a better overview.
+At 9.15 a.m. we have our Scrum Master's Daily Scrum where I exchange with the other Scrum Master from Hamburg, for example about ongoing topics in the teams.
+Half an hour later I have the Daily Scrum with my teams.
+After the Daily Scrum and before my lunch break, I try to work on my tasks which I priotized in the morning.
+
++ **Lunch time:**
+I usually spend my lunch break together with the team.
+Sometimes we meet in our social room to eat together, or go outside to have lunch.
+On other days we even cook together.
+For example, this week I will prepare Kaiserschmarrn (since I am originally from Austria) for the whole team. 
+
+* **Afternoon:**
+The afternoon often differs, depending on if we have a Sprint change or not.
+If we have a Sprint change, I spend the day with planning Reviews, and Retrospectives.
```

Comment:
„... change, I spend most of the day with meetings (e.g. Sprint Review, Retrospective and Sprint Planning).“